src/components/LogRow/BaseRow/index.tsx

```diff
@@ -235,6 +235,10 @@ const Index = styled.pre`
   line-height: inherit;
   font-size: inherit;
   user-select: none;
+
+  ::before {
+    ${({ lineNumber }) => lineNumber && `content: "${lineNumber}";`}
+  }
```

Comment:
The truthiness check is preventing `0` from being printed on the first line:
```suggestion
  ::before {
    ${({ lineNumber }) => `content: "${lineNumber}";`}
  }
```

<img width="403" alt="Screenshot 2023-03-17 at 3 16 18 PM" src="https://user-images.githubusercontent.com/47064971/225998926-770f76e5-9feb-4da8-a2b1-e8bf30d9678e.png">

Reply:
Great catch!